internal/command/views/modules.go

```diff
@@ -38,9 +39,34 @@ type ModulesHuman struct {
 var _ Modules = (*ModulesHuman)(nil)
 
 func (v *ModulesHuman) Display(manifest moduleref.Manifest) int {
+	if len(manifest.Records) == 0 {
+		v.view.streams.Println("No modules found in configuration.")
+		return 0
+	}
+	printRoot := treeprint.New()
+	populateTreeNode(printRoot, &moduleref.Record{
+		Children: manifest.Records,
+	})
+
+	v.view.streams.Println(fmt.Sprintf("Modules declared by configuration:\n\n%s", printRoot.String()))
```

Comment:
Output should be as follows (we want consistency with `terraform providers`):
```
$ terraform modules
\n
Modules declared by configuration:
\n
... display tree
```

Reply:
Included 👍 https://github.com/hashicorp/terraform/pull/36062/commits/9a3cedb499c6bb660539bf0e8b5a87ddb1e796ce